app/views/layouts/_filter_select.html.haml

```diff
@@ -0,0 +1,7 @@
+= f.input filter.searchable_name, collection: filter.input_options, label: filter.input_label,
+                                  selected: search.selected_value(filter.searchable_name), include_blank: filter.blank_value_label,
+                                  as: filter.input_type, group_method: :last, required: false, label_method: filter.label_method,
+                                  input_html: { class: 'selectpicker',
+                                                multiple: filter.multiple,
+                                                'data-live-search' => true,
```

Comment:
can we correct the hash syntax? this should work?: `'data-live-search': true`

Reply:
there are more instances of the hash syntax, pls do a search

Reply:
fixed